### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GilbertMiao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
   4. test case: bad pr, commits after comments; he said need change, so i change.
   5. no comments at all
   6. bad pr
+  7. good pr
```
Comment (pzckwoop) [inline README.md]: change required

**Review by pzckwoop — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GilbertMiao) [reply README.md]: ok

**Comment by GilbertMiao** [comment]
@pzckwoop plz reivew again

**Comment by pzckwoop** [comment]
ok to me

**Pull request merged**